src/screens/APIUtils/APIUtils.res

```diff
@@ -266,8 +267,8 @@ let handleLogout = async (
   ~setIsSidebarExpanded,
   ~clearRecoilValue,
 ) => {
-  // let logoutUrl = getURL(~entityName=USERS, ~methodType=Post, ~userType=#SIGNOUT, ())
-  // let _ = await fetchApi(logoutUrl, ~method_=Fetch.Post, ())
+  let logoutUrl = getURL(~entityName=USERS, ~methodType=Post, ~userType=#SIGNOUT, ())
+  let _ = await fetchApi(logoutUrl, ~method_=Fetch.Post, ())
```

Comment:
do we need await?

Reply:
yes, because just below that we are clearing the local storage but token is required to make this api call and hence execute it succesfully.

Reply:
Just add a try catch if any error comes